content/en/universal_service_monitoring/setup.md

```diff
@@ -760,6 +760,51 @@ DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_ENABLED=true
 
 {{< /tabs >}}
 
+### Go TLS Monitoring
+
+<div class="alert alert-info">
+Universal Service Monitoring is available in <strong>beta</strong> to monitor TLS encrypted traffic from services implemented in Golang.
+
+<i>Note: Go HTTPs servers can upgrade HTTP1.1 protocol to HTTP/2 which is supported in private beta, please reach to your account manager for further details<i>
```

Comment:
The text is awkwardly sized here -- I think it's fine to move this note out of this alert and alongside the agent version requirement.

<img width="734" alt="Screenshot 2024-03-07 at 7 28 41 AM" src="https://github.com/DataDog/documentation/assets/295551/5df20729-44fd-4735-aa8f-ecb45d65b8bf">